daemon/openrazer_daemon/dbus_services/dbus_methods/deathadder_chroma.py

```diff
@@ -81,6 +81,8 @@ def get_logo_effect(self):
     :return: Active
     :rtype: bool
     """
+    # TODO: remove. we have a new solution that works across all
+    # devices and is more reliable (getCurrentLogoEffect)
```

Comment:
Could you grep through the applications listed in the readme to see if this method gets used?

Reply:
Polychromatic: Not used
razerCommander: Not used
Chroma Feedback: Not used
RazerGenie: Method exposed in libopenrazer, otherwise not used

Bonus! KeyboardVisualizer: Not used

Reply:
I will have to tweak the Kraken device suspend code in order to remove the Kraken-specific getCurrentEffect method...

Reply:
Done